c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -70,14 +70,23 @@ Version 1.6
     + add_interface(config, 'routing')
     + add_interface(config, 'mobile', INTERFACE_TYPE_SENCHA_TOUCH)
 
-    In the `add_static_view` do the following change:
+    Remove the `add_static_view`:
     - # add the main static view
     - config.add_static_view(
     -     name='proj',
     -     path='{{package}}:static',
     -     cachebust=True,
     - )
-    + add_static_view(config)
+
+    Remove the adding of the admin interface:
```

Comment:
Remove the admin interface activation switch (now done directly in c2cgeoportal):